src/common/components/layouts/pages/inspect/validationErrorStatus.json

```diff
@@ -211,6 +211,101 @@
         "STATUS": "warning",
         "CATEGORY": "structure",
         "RATIONALE": "Lists should not be misused to represent tables."
+      },
+      "27": {
+        "STATUS": "warning",
+        "CATEGORY": "structure",
+        "RATIONALE": "Multiple paragraphs should be tagged as such. Line breaks should not be used to delimit paragraphs."
+      },
+      "28": {
+        "STATUS": "warning",
+        "CATEGORY": "structure",
+        "RATIONALE": "Table of content items should be tagged as such."
+      },
+      "29": {
+        "STATUS": "warning",
+        "CATEGORY": "structure",
+        "RATIONALE": "Table of content items should not be empty."
+      },
+      "30": {
+        "STATUS": "warning",
+        "CATEGORY": "code",
+        "RATIONALE": "Table of content items should point somwhere in the document."
+      },
+      "31": {
+        "STATUS": "warning",
+        "CATEGORY": "code",
+        "RATIONALE": "Table of content items should point to the right place in the document."
+      },
+      "32": {
+        "STATUS": "warning",
+        "CATEGORY": "structure",
+        "RATIONALE": "Table of content items should be correctly aligned for readability."
+      },
+      "33": {
+        "STATUS": "warning",
+        "CATEGORY": "structure",
+        "RATIONALE": "Table of contents item should match the heading it refers to."
+      },
+      "34": {
+        "STATUS": "warning",
+        "CATEGORY": "structure",
+        "RATIONALE": "A single list should be tagged as such and not a series of smaller lists."
```

Comment:
It may be not tagged as several lists. It could be several paragraphs, spans, or else structure elements.